Runtime/API.cs

```diff
@@ -45,6 +45,10 @@ public static async UniTask<LoginPayload> RequestLogin(string walletAddr, Cancel
 
         public static async UniTask<LoginResponse> Login(LoginRequest loginRequest, CancellationToken token = default)
         {
+            if (!WebEventManager.FingerprintAvailable())
+            {
+                Debug.Log("WARNING: Fingerprint Data Not Yet Available");
```

Comment:
at some point we prob want to have a conditional debugging mode.

Reply:
agreed